server/modules/analysis/service/rChainService/index.js

```diff
@@ -6,14 +6,17 @@ import * as CSVReader from '@server/utils/file/csvReader'
 import * as Survey from '@core/survey/survey'
 import * as NodeDef from '@core/survey/nodeDef'
 import * as ProcessingStep from '@common/analysis/processingStep'
+import * as ProcessingStepCalculation from '@common/analysis/processingStepCalculation'
 import * as ResultStepView from '@common/surveyRdb/resultStepView'
 
 import * as SurveyManager from '@server/modules/survey/manager/surveyManager'
 import * as SurveyRdbMamager from '@server/modules/surveyRdb/manager/surveyRdbManager'
 
 import * as ProcessingChainManager from '../../manager/processingChainManager'
+import * as CalculationManager from '../../manager/calculation'
```

Comment:
CalculationManager must not exist. Only AnalysisManager

Reply:
ok